theme/skip-links/_base.scss

```diff
@@ -9,3 +9,9 @@
 		text-decoration: none;
 	}
 }
+
+button.wb-sl:focus {
+	border: none;
+	margin: 0;
+	width: auto;
+}
```

Comment:
I think this change is not related at this PR subject. It will probably be best to either move this in a separate commit or do a different pull request